src/CommonITILSatisfaction.php

```diff
@@ -286,7 +297,9 @@ public static function displaySatisfaction($value, $entities_id)
      * Get name of inquest type
      *
      * @param int $value Survey type ID
-     **/
+     *
+     * @return string|int
```

Comment:
Maybe the default case should return `(string) $value`to enforce the string type.

```suggestion
     * @return string
```